src/styles/blocks/watch.scss

```
@@ -0,0 +1,61 @@
+.watch {
+  position: relative;
+  background-color: white;
+  border: 15px solid rgb(255, 255, 255);
+  border-radius: 50%;
+
```

to align the arrows strictly in the center
```suggestion
  position: relative;
  background-color: white;
  border: 15px solid rgb(255, 255, 255);
  border-radius: 50%;
	
	display: flex;
  justify-content: center;
  align-items: center;
```